test/modules/Relayer/Program.cs

```diff
@@ -52,6 +53,18 @@ static async Task Main(string[] args)
             }
         }
 
+        static Task ExecuteWithRetry(Func<Task> func, Action<RetryingEventArgs> onRetry)
+        {
+            var transientRetryPolicy = RetryPolicy.DefaultExponential;
+            transientRetryPolicy.Retrying += (_, args) => onRetry(args);
+            return transientRetryPolicy.ExecuteAsync(func);
+        }
+
+        static void RetryingReportTestResult(RetryingEventArgs retryingEventArgs)
```

Comment:
This method name should change; should be something related to logging retry event.